src/Symfony/Component/Asset/PathPackage.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset;
+
+use Symfony\Component\Asset\VersionStrategy\VersionStrategyInterface;
+
+/**
+ * Package that adds a base path to asset URLs in addition to a version.
+ *
+ * In addition to the provided base path, this package also automatically
+ * prepends the current request base path if a Context is available to
+ * allow a website to be hosted easily under any given path under the Web
+ * Server root directory.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class PathPackage extends Package
```

Comment:
I'd consider decorating the `PackageInterface` instead of extending the `Package`. This way it would be possible to prepend the base path to any `PackageInterface` implementation.